### DISCUSSION TITLE
How to run Renovate against a subset of repositories based on a hashing logic or something similar?

### DISCUSSION THREAD

**Discussion by phaninra** [renovatebot/renovate#26304] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

34.4.0

### If you're self-hosting Renovate, select which platform you are using.

gitlab.com

### What is your question?

Let's assume there's 100 target repositories to renovate.  I'd like Renovate to run each day but only process 10% each day, i.e. 10 days to renovate all of them and on the 11th day Renovate should process the same ones it processed on Day 1. The number '10' is just an example here. It could be based on day of the month bucket, etc. 

How can I achieve something like this without having to add some setting in each repo's renovate.json file? 

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by phaninra** [comment]
Wondering if anyone had any ideas? Thank you.

**Reply by secustor** [reply]
You can program your hashing logic in `config.js` as you self-host

**Reply by phaninra** [reply]
Thanks for the response. I actually use the https://gitlab.com/renovate-bot/renovate-runner  which I didn't see (might have missed it) in the dropdown while posting this. Would your comment still apply?

**Reply by secustor** [reply]
Yes, this still applies.

Depending on your repo setup you have to overwrite the config location.

**Reply by phaninra** [reply]
(a backend dev trying to figure this out)

There's a config.js in the setup which sets `autodiscoverFilter` with a list of `projectGroup1/*, projectGroup2/*, etc.`.  Could you please provide a bit more info on how to go about filtering a repo based on hashing logic? I looked into available configuration attributes and couldn't find anything relevant. Thanks again.
